.examples/raspberrypi/docker-compose.yml

```diff
@@ -0,0 +1,42 @@
+version: "3.4"
+
+services:
+  app:
+    image: monica
+    env_file: .env
+    depends_on:
+      - db
+    ports:
+      - 8080:80
+    volumes:
+      - data:/var/www/html/storage
+    restart: always
+
+  cron:
+    image: monica
+    env_file: .env
+    restart: always
+    volumes:
+      - data:/var/www/html/storage
+    command: cron.sh
+    depends_on:
+      - db
```

Comment:
We should not mix the example. Here it's for raspberry, and the `cron` part is not mandatory, please remove it from the example.
```suggestion
```

Reply:
I believe the `cron` is mandatory: without it, reminder emails are not sent (and therefore the application setup isn't complete). Am I missing something here? Is there another way for reminders and other async jobs to run?>

Reply:
@asbiin friendly reminder on my question about async jobs running when cron is not in place. Thanks!

Reply:
@iloveitaly I stumbled upon the same issue and was about to give the cron thing a try, but looking further it seems that the supervisor method is the way to go: https://github.com/monicahq/docker/blob/master/.examples/supervisor/fpm/docker-compose.yml 

and you need to copy the files from https://github.com/monicahq/docker/tree/master/.examples/supervisor/fpm/app as mentioned in the docs, once it was running with supervisor the reminder mails stared coming in as expected.

Reply:
@lcx why is the supervisor method the way to go? Running cron in a separate container doesn't seem like a bad option, and has some benefits compared to the supervisor method (more configuration i docker vs lesser known unix tooling; easier monitoring and log analysis of separate processes; etc).

Reply:
@asbiin the cron part is mandatory if you want reminder emails and other async jobs to be run, right?

Reply:
@iloveitaly indeed, the cron part is here to run the schedule command, which aims at sending reminders, and other tasks. It's not mandatory to run the application, that's why it's not included in all examples, also because there are multiple ways to handle it.

Reply:
Also I agree with @iloveitaly : the "normal" way do create cron part with docker is running another container, not using supervisor. Each container has its own purpose, and it's easier to start/stop only one without touching the others.

Maybe it could be better to have a dedicated section about the cron/scheduler part - I would agree documentation is not the best part of this project!

Reply:
@asbiin what are your thoughts about merging this in and handling the future improvements to documentation in a separate PR? Having this example in this repo would have been super helpful for me as I was trying to host this on my pi.

Reply:
@asbiin friendly reminder on my last comment: thoughts on merging as-is and then tweaking later?